File: examples/javascript/constructs/scope_shadowing.js
// Ported from examples/lumen/constructs/scope_shadowing.lm by scripts/port_examples.py; edit the Lumen original, not this file.
console.log(1);
let x = 10;
console.log(x);
if (true) {
    x = 20;
    console.log(x);
}
console.log(x);

File: examples/rust/constructs/scope_shadowing.rs
// Ported from examples/lumen/constructs/scope_shadowing.lm by scripts/port_examples.py; edit the Lumen original, not this file.
fn main() {
    println!("{}", 1);
    let mut x = 10;
    println!("{}", x);
    if true {
        x = 20;
        println!("{}", x);
    }
    println!("{}", x);
}
